### ISSUE TITLE
Conv Details

### ISSUE THREAD

**Issue by hhqweasd** [martkartasev/sepconv#23]
The conv in original is
==>Basic()
==>torch.nn.Conv2d(in_channels=intInput, **out_channels=intOutput,** ...)
Yours is
==>_conv_module()
==>torch.nn.Conv2d(in_channels, **in_channels,** kernel, stride, padding)
The channel is different.

**Comment by xiaozhi2015** [comment]
After fixing this bug, have you obtained the result same as the original author's  [https://github.com/sniklaus/sepconv-slomo](url) ?
I just got the blur result in this project.
